apps/condo/domains/notification/utils/serverSchema/index.js

```diff
@@ -151,6 +152,21 @@ async function checkMessageTypeInBlackList (context, message) {
         if (isMessageTypeInUserBlackList) return { error: MESSAGE_TYPE_IN_USER_BLACK_LIST }
     }
 
+    if (get(message, ['meta', 'data', 'B2CAppId'])) {
```

Comment:
it's not a part of this domain logic. You don't need to change anything inside the notification domain!

Reply:
Move it inside the miniapp domain